tests/helpers/TestBackupChecker.py

```diff
@@ -3,6 +3,7 @@
 # -*- coding: utf-8 -*-
 import json
 from base64 import b64encode
+from typing import cast
```

Comment:
This file should be renamed to TestBackupHelper, but tbh Helper doesn't say anything about what the class should do..

Reply:
Maybe BackupManager is a better suited name?